src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -8,13 +8,10 @@
 
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import hudson.EnvVars;
-import hudson.FilePath;
-import hudson.Launcher;
+import hudson.*;
```

Comment:
Please don't switch to wildcard imports.  That change tends to result in unnecessary conflicts with other pull requests and forces work onto the authors of other pull requests.

I'll include a pointer to the SCM API guidelines in the CONTRIBUTING file so that others are less likely to have that problem

Reply:
Sorry, fixed now.